### PR TITLE
ref(conn-indicator): remove E2E RTT

### PR DESCRIPTION
**Pull request by mykhailo-ua**

### CODE REVIEW

**Comment by saghul** [comment]
Why?

**Comment by paweldomas** [comment]
There was a fundamental change in how e2e pings work in lib-jitsi-meet where it does few pings at the beginning, pushes metrics and shuts the whole thing down, so there are no more e2ertt updates after that. Plus it's disabled by default. We have decided with @bgrozev to remove that instead of trying to adjust. I think even if e2e pings are enabled the data is not feed correctly to the indicators.

File: react/features/rtcstats/RTCStats.ts [hunk 1]
```diff
@@ -121,16 +120,6 @@ class RTCStats {
         this.trace && this.trace.statsEntry('dominantSpeaker', null, dominantSpeakerData);
     }
 
-    /**
-     * Send e2e rtt data, the data will be processed by rtcstats-server and saved in the dump file.
-     *
-     * @param {Object} e2eRttData - The object that holds the e2e data.
-     * @returns {void}
-     */
-    sendE2eRttData(e2eRttData: E2ERTTData) {
```
Comment (paweldomas) [inline react/features/rtcstats/RTCStats.ts:130]: @mykhailo-ua I think we want to keep the analytics and stats part, just the UI needs to go away.

Comment (mykhailo-ua) [reply react/features/rtcstats/RTCStats.ts]: reverted that part.

**Comment by saghul** [comment]
SGTM but I think this info needs to be in the commit message so it's not lost :-)

**Comment by bgrozev** [comment]
LGTM

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by saghul** [comment]
Can you please rebase and fix the conflict @mykhailo-ua ?

**Review by paweldomas — APPROVED** [review]

**Pull request merged**